docs/source/06_nodes_and_pipelines/04_micro_packaging.md

```diff
@@ -0,0 +1,124 @@
+
+# Micro-packaging
+
+Micro-packaging allows users to share Kedro pipelines across codebases, organisations and beyond.
+
+## Package a modular pipeline
+
+Since Kedro 0.16.4 you can package a modular pipeline by executing:
```

Comment:
```suggestion
Since Kedro 0.16.4 you can package a modular pipeline by executing:
```

I think we can remove the "since 0.16.4" because it's sufficiently old now, and it might confuse users in thinking the rest of the feature (micropackaging) is also available since then.